test/built-ins/Temporal/TimeZone/prototype/equals/argument-object.js

```diff
@@ -0,0 +1,56 @@
+// Copyright (C) 2020 Igalia, S.L. All rights reserved.
```

Comment:
The copyright should say 2023 and belong to you, since you contributed them (or, in the general case, to your employer if your contract with them requires it) — even if they are similar to other tests elsewhere in the suite.

Reply:
OK, will do. Didn't even notice the copyright, lol. Too busy copy/pasting boilerplate.